### PR TITLE
Adding Initial support for Durgod Taurus K320 TKL

### PR DESCRIPTION
**Pull request by kuenhlee**
## Description
Adding QMK support to Pre-built Durgod Taurus K320

- Added Generic default keymap
- Added User specific keymap
- Updated readme.md

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/durgod_k320/config.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+#define TKL_DURGOD_K320
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xDEAD
+#define PRODUCT_ID      0xFACE
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    Durgod-QMK 
+#define PRODUCT         K320
+#define DESCRIPTION     "A QMK Enabled Durgod TKL"
```
[the remark this answers is not in the document]
Comment (kuenhlee) [reply keyboards/durgod_k320/config.h:30]: - just curious what wrong with the description.
- If to change, I will also suggest to change the manufacturer to just Durgod.

**Comment by tzarc** [comment]
Did you replicate the entirety of `boards/ST_NUCLEO64_F070RB` just to modify `STM32_HSECLK`?

File: keyboards/durgod_k320/config.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (tzarc) [reply keyboards/durgod_k320/config.h:30]: As per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist), `DESCRIPTION` is no longer required.

**Comment by kuenhlee** [comment]
I don't want to touch anything in the core and just picked a existing example in the master branch to follow through similar changes especially there are no documentation taking care of this type of flows

**Comment by tzarc** [comment]
> I don't want to touch anything in the core and just picked a existing example in the master branch to follow through similar changes especially there are no documentation taking care of this type of flows

I asked because I need to know what modifications were done, so that I can tell you what you need to do in order to remove the board files.

**Comment by reecefowell** [comment]
@kuenhlee why did you close this? I also own a Durgod K320 and these changes could be very useful to me and others no doubt. Are there issues with this PR that can be resolved? Thanks.

**Pull request closed without merging**